reports/variant_report_latest.md

```diff
@@ -32,42 +33,57 @@ Nonetheless, the apparent growth advantage of BA.2.75 over BA.5\* at whole count
 
 ![picture 5](../images/a0891368449cb6070ab832591937ace96d6b3ef704f7a3fa17e902a1c33ea035.png)
 
-An up to date version of this graph can be found here: <https://cov-spectrum.org/explore/India/AllSamples/Past3M/variants?variantQuery=NextcladePangoLineage%3ABA.5*&variantQuery1=NextcladePangoLineage%3ABA.2.75&analysisMode=CompareToBaseline&>
+An up to date version of this graph can be found [here](https://cov-spectrum.org/explore/India/AllSamples/Past3M/variants?variantQuery=NextcladePangoLineage%3ABA.5*&variantQuery1=NextcladePangoLineage%3ABA.2.75&analysisMode=CompareToBaseline&)
 
 ### Selection of other interesting lineages
 
 Many interesting BA.2 sublineages have appeared first in India. Here is a selection of some to watch. That mutations are typical of BA.2 sublineages with (potential) growth advantages, in India and also globally.
 
 #### BA.2.76 (S:R346T, S:Y248N)
 
+As of **DATE** there are approximately 1,800 sequences worldwide, and growth advantage is currently estimated at 36%. The first sample is from March.
+
+The Pango designation issue, with discussion about this lineage:
 <https://github.com/cov-lineages/pango-designation/issues/787>
 
+An up-to-date CoV-Spectrum summary of this lineage:
 <https://cov-spectrum.org/explore/World/AllSamples/Past3M/variants?pangoLineage=BA.2.76>
 
 #### BA.2.38.1 (S:N417T, S:S247N, S:Y248S, S:K444N)
 
+As of **DATE** there are approximately 192 sequences worldwide, and growth advantage is currently estimated at 47%. The first sample was detected mid-May.
+
+The Pango designation issue, with discussion about this lineage:
 <https://github.com/cov-lineages/pango-designation/issues/809>
 
+An up-to-date CoV-Spectrum summary of this lineage:
 <https://cov-spectrum.org/explore/World/AllSamples/Past3M/variants?pangoLineage=BA.2.38.1>
 
 #### BA.4.6 (S:346T)
 
+As of **DATE** there are approximately 2,500 sequences worldwide, and growth advantage is currently estimated at 66%. The first samples were detected at the end of April.
+
+An up-to-date CoV-Spectrum summary of this lineage:
 <https://cov-spectrum.org/explore/World/AllSamples/Past6M/variants?aaMutations=S%3AR346T%2CN%3A151S%2CORF7b%3AL11F&nucMutations=12160A%2CC8140T>
 
 ## Latest developments
 
 ### South African pair of complex recombinant clusters
 
-Two related clusters of complex Delta/BA.2 recombinants have appeared in Gauteng/Limpopo (bordering region).
```

Comment:
How many sequences are in each of these clusters?  They are called C1 and C2 right? (From figures below?)